site/docs/try/configuration.md

```diff
@@ -234,30 +231,22 @@ Metrics are published using prometheus and can be collected via standard methods
 Since Nessie 0.46.0, traces are published using OpenTelemetry. See [Using
 OpenTelemetry](https://quarkus.io/guides/opentelemetry) in the Quarkus documentation.
 
-In order for the server to publish its traces, the
-`quarkus.otel.exporter.otlp.endpoint` property _must_ be set. Its value must be a
+In order for the server to enable OpenTelemetry and publish its traces, the
+`quarkus.otel.exporter.otlp.traces.endpoint` property _must_ be defined. Its value must be a
```

Comment:
I believe the logic in this PR will enable OTel if either of these properties is set.

Reply:
Yes, but although Quarkus recognizes both, it encourages users to use `quarkus.otel.exporter.otlp.traces.endpoint`. That's why I didn't mention the other one. Do you prefer to have both mentioned here?

Reply:
If `quarkus.otel.exporter.otlp.traces.endpoint` is the primary config, let's us it.

I am basically worried about confusing end users :thinking: If Quarkus does not effectively use `quarkus.otel.exporter.otlp.endpoint`, do we need to check it?